ste/jobStatusManager.go

```diff
@@ -155,8 +176,19 @@ func (jm *jobMgr) handleStatusUpdateMessage() {
 		case <-jstm.listReq:
 			/* Display stats */
 			js.Timestamp = time.Now().UTC()
-			jstm.respChan <- *js
-
+			if jstm.respChan != nil {
+				select {
+				case jstm.respChan <- *js:
+					// Send on the channel
+				case <-jstm.statusMgrDone:
+					// If we time out, no biggie. This isn't world-ending, nor is it essential info. The other side stopped listening by now.
+				}
+				defer func() { // Exit gracefully if panic
```

Comment:
Should this code come before reading from respChan?

Reply:
A read won't panic; this is probably unnecessary.